### DISCUSSION TITLE
NoSQL database options are empty.

### DISCUSSION THREAD

**Discussion by lohxx** [dbeaver/dbeaver#17644] [Q&A]
### Description

<img width="1440" alt="Captura de Tela 2022-08-22 às 21 19 23" src="https://user-images.githubusercontent.com/26140123/186042236-9e3e502a-658e-403b-9a79-8f483df204fd.png">

### DBeaver Version

22.1.5

### Operating System

macOs Monterey 12.5.1

### Steps to reproduce

1. Click on the button to connect a database
2. Select the NoSQL option on the left.

**Comment by Destrolaric** [comment]
Hi, could you please send me the log files? You can find instructions here: https://dbeaver.com/docs/wiki/Log-files/. Thank you.
Edit. Never mind, I confirm that the NoSQL tab should not be visible on CE.

**Comment by serge-rider** [comment]
NoSQL databases supported only in PRO version of DBeaver.
